extpsdk/src/main/python/vantiqconnectorsdk.py

```diff
@@ -220,7 +223,15 @@ def __init__(self, source_name: string, config: Union[dict, None]):
             if kwArgString is not None:
                 try:
                     kw_temp : dict = json.loads(kwArgString)
-                    self.connect_kw_args = kw_temp
+                    disable_ssl = kw_temp.get(VantiqConnector.DISABLE_SSL_VERIFICATION)
+                    if disable_ssl is not None and disable_ssl:
+                        self.disable_ssl_check = True
+                        ctx = ssl.create_default_context()
+                        ctx.check_hostname = False
+                        ctx.verify_mode = ssl.CERT_NONE
+                        self.connect_kw_args = {"ssl": ctx}
```

Comment:
is there any use case for merging this SSL setting with other settings that might be in kw_temp (minus the DISABLE_SSL_VERIFICATION setting) ?

Reply:
We could do that.  It doesn't look like the others, but having it available isn't a bad thing.  So far, I just rerun locally and find issues that way, but having it around wouldn't hurt.

Reply:
There may be.  But I didn't want to deal with conflict resolution, etc., at this point, so I figured we could wait and see if there's any demand.  This is the first time this has come up, and it's primarily, in that case, an issue with the client environment.  But it's useful for dealing with installations with self-signed certs, so it seemed worthy of adding.